### ISSUE TITLE
App starts on featured channels on AAOS, not good

### ISSUE THREAD

**Issue by Raitsa** [ChernyshovYuriy/OpenRadio#38]
With Android Automotive OS, the app now starts on Featured stations which is not good while starting to drive the car, needing the driver to focus on changing to radio channel to a favourite. 

Please let it start on favourites (or have this as an option) like it used to do before Featured channels were available.

**Comment by Jerther** [comment]
Yes please I was planning on reporting this as well. 

I might also add that the home screen's "For you" proposes stuff from the Feature tab, which I really don't care about. It'd be nice if it presented stations from favorites. Or at least the few last played stations.

**Comment by ChernyshovYuriy** [comment]
While I understand your concerns, I afraid I dont have a timeline for that.
Google now demand 2 items from me to submit before they allow me to submit updates:
- a video indicating how permission to start Open Radio service for Media from background works !! This is new permission they introduce and force to use for media apps and yet they require to provide a video how it works!
- Since I use Login option also provided by Google to back up Favorites, Google asks to provide a link which users can use to delete account.

Since Open radio is a bit old project (since 2014) and I dont have much tie as I had 10 years ago for a project that dont provide any income, I am not sure when I will be able to please Google with demands :-( 

![Screenshot from 2023-12-18 13-36-32](https://github.com/ChernyshovYuriy/OpenRadio/assets/931955/f7afcae1-cafc-4982-87e5-b6eb591ed7a0)
![Screenshot from 2023-12-18 13-37-07](https://github.com/ChernyshovYuriy/OpenRadio/assets/931955/98366214-7de9-4122-ba18-65bc260be08c)

**Comment by ChernyshovYuriy** [comment]
This is mostly the reason I dont like any "features": Logs, Location, Login /Password, etc ... it gives too much headache, if not not now then later for sure. Google creates obstacles once in a year for devs, base don my decade experience.

**Comment by Jerther** [comment]
Do you have some analytics on the popularity of your project? I learned its existance from a thread on reddit. It may gather enough popularity to attract some donations if you need them.

That being said, I really appreciate the efforts you put into OpenRadio. Thank you :)

**Comment by ChernyshovYuriy** [comment]
I have a lot of data from Google analytics, all crashes, minor issues, users ... Google has enormous amount of data.
Daily, Open Radio has about 8K users, not too bad, and the Donation Button is in About section but it is used by ~0.5 % of users, almost no one. I have got maybe dozens of donations overall. Which is still make me happy.
The point is, I got radio stations from the free and open database and I have no right for the streaming content, so I can charge for that or even introduce advertisement, which make it unique I would say.
I am more disappointment of Google. They introduced that permission and now asks me to provide video to prove how I use it :-( 
....
While I still like Open Radio I am looking for other opportunities in mobile app world.
I am working on BrewXperience now - a mobile app based on Google data to highlight craft breweries around the location of the device.

**Comment by Jerther** [comment]
I haven't noticed the donate button. Here is a thirteenth one 😉

I really like open radio and I wouldn't mind side loading it if it means you don't have to make videos every update

**Comment by ChernyshovYuriy** [comment]
Good idea, I can side load [here](https://github.com/ChernyshovYuriy/OpenRadio/tree/master/app/store).
But I need to solve the issue with Google, the initial and still the current intention is to bring Radio to vehicle which this app does - Android Auto and more important Android Automotive OS. Delivery tho these markets is only through the Store.

**Comment by ChernyshovYuriy** [comment]
Ok, seems like Google accepted my video about permission and Delete Account explanation :-) 
I also changed Featured tab for Android Automotive.

**Comment by Jerther** [comment]
I just realized, AAOS in the original post is AA Automotive, not AA 😥 I wanted to report this issue but for AA.
